21. Svetainių talpinimo platformos: Vercel ir Netlify praktinis naudojimas.md

```diff
@@ -0,0 +1,195 @@
+# Svetainės Paskelbimas Vercel Platformoje
```

Comment:
visus title sutvarkyti reikia, kad butu is mazosios raides `Svetaines paskelbimas ...`